featuretools/primitives/base/aggregation_primitive_base.py

```diff
@@ -25,6 +25,15 @@ def generate_name(self, base_feature_names, relationship_path_name,
             self.get_args_string(),
         )
 
+    def generate_names(self, base_feature_names, relationship_path_name,
+                       parent_entity_id, where_str, use_prev_str):
+        n = self.number_output_features
+        return [self.generate_name(base_feature_names,
```

Comment:
this will call `self.generate_name` n times, let's store the result of the call in a variable instead